includes/admin/tools/class-settings-import.php

```diff
@@ -0,0 +1,681 @@
+                         data-end="<?php echo $index_end; ?>"
+                         data-next="<?php echo $next; ?>"
+                         data-total="<?php echo $total; ?>"
+                         data-per_page="<?php echo self::$per_page; ?>">
+
+                        <div style="width: <?php echo $current_percentage; ?>%"></div>
+                    </div>
+                    <input type="hidden" value="3" name="step">
+                    <input type="hidden" value='<?php echo maybe_serialize( $_REQUEST['mapto'] ); ?>' name="mapto"
+                           class="mapto">
+                    <input type="hidden" value="<?php echo $_REQUEST['csv']; ?>" name="csv" class="csv">
+                    <input type="hidden" value="<?php echo $_REQUEST['mode']; ?>" name="mode" class="mode">
+                    <input type="hidden" value="<?php echo $_REQUEST['create_user']; ?>" name="create_user"
+                           class="create_user">
+                    <input type="hidden" value="<?php echo $_REQUEST['delimiter']; ?>" name="delimiter">
+                    <input type="hidden" value='<?php echo maybe_serialize( self::get_importer( $csv ) ); ?>'
+                           name="main_key"
+                           class="main_key">
+                </th>
+            </tr>
+
+            <script type="text/javascript">
+                jQuery(document).ready(function ($) {
+                    give_on_donation_import_start();
+                });
+            </script>
+			<?php
+		}
+
+		/**
+		 * Will return true if importing can be started or not else false.
+		 */
+		static function check_for_dropdown_or_import() {
+			$return = true;
+			if ( isset( $_REQUEST['mapto'] ) ) {
+				$mapto = (array) $_REQUEST['mapto'];
+				if ( false === in_array( 'form_title', $mapto ) && false === in_array( 'form_id', $mapto ) ) {
+					Give_Admin_Settings::add_error( 'give-import-csv-form', __( 'Please select Form ID or Form Name options from the dropdown.', 'give' ) );
+					$return = false;
+				}
+
+				if ( false === in_array( 'amount', $mapto ) ) {
+					Give_Admin_Settings::add_error( 'give-import-csv-amount', __( 'Please select Amount option from the dropdown.', 'give' ) );
+					$return = false;
+				}
+
+				if ( false === in_array( 'email', $mapto ) && false === in_array( 'customer_id', $mapto ) ) {
+					Give_Admin_Settings::add_error( 'give-import-csv-donor', __( 'Please select Email id or Customer ID options from the dropdown.', 'give' ) );
+					$return = false;
+				}
+			} else {
+				$return = false;
+			}
+
+			return $return;
+		}
+
+		/**
+		 * Print the Dropdown option for CSV.
+		 */
+		static function render_dropdown() {
+			$csv = (int) $_GET['csv'];
+			?>
+            <tr valign="top" class="give-import-dropdown">
+                <th colspan="2">
+                    <h2 id="give-import-title"><?php esc_html_e( 'Map CSV fields to donations', 'give' ) ?></h2>
```

Comment:
@raftaar1191 remove unnecessary `esc_html` from hard coded strings

Reply:
@ravinderk 

I was reading this article that why I used it. Please have a look into it
https://vip.wordpress.com/documentation/validating-sanitizing-escaping/#always-escape-late